app/components/user-status.js

```diff
@@ -0,0 +1,33 @@
+import Component from '@glimmer/component';
+
+export default class UserStatusComponent extends Component {
+  currentUserStatus = [
+    {
+      status: 'active',
+      message: 'I am doing a task',
+      update: 'idle',
+      class: 'buttons__idle',
+      buttonText: 'Change status to ‘Idle’',
+      update2: 'ooo',
+      buttonText2: 'Mark yourself as OOO',
```

Comment:
Can we have better names for `update`, `update2` etc?
They don't really tell me anything about what they're supposed to do, unless I see the code and the design

Reply:
Updated